src/mux-analytics.brs

```diff
@@ -904,6 +906,34 @@ function muxAnalytics() as Object
     m._addEventToQueue(m._createEvent("rebufferend"))
   end sub
 
+  prototye.playbackModeHandler = sub(playbackMode as Object)
+    props = {}
+
+    if playbackMode.mode = Invalid
+      print "[mux-analytics] warning: playback_mode mode property not set."
+      return
+    end if
+    props.player_playback_mode = playbackMode.mode
+
+    if playbackMode.player_playback_mode_data = Invalid
```

Comment:
Is this what we want to specify as the field name, @daytime-em ? It feels a little bit verbose, and I'm not really _against_ it, but if we have `playbackMode.mode` and `playbackMode.data` wouldn't that work for us?